app/Kconfig

```diff
@@ -102,7 +102,6 @@ menuconfig ZMK_BLE
 	select BT_SMP_APP_PAIRING_ACCEPT
 	select BT_PERIPHERAL
 	select BT_DIS
-	select BT_BAS
```

Comment:
Maybe worth trying instead:

```
        select BAS if !ZMK_SPLIT_ROLE_CENTRAL
```

Reply:
```
  select BT_BAS if !ZMK_SPLIT_ROLE_CENTRAL
```

Required to build the non-central side of the keyboard